<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>Time
Sloc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="timeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="labelMoney">
        <property name="text">
         <string>Money</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="moneyEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="labelKarma">
        <property name="text">
         <string>Company
Karma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="karmaEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="buttonPersonal">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="toolTip">
         <string>personal</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>applications.png</normaloff>applications.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTool">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>program.png</normaloff>program.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStat">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>stat.png</normaloff>stat.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAnzeige">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Overview Company</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>persons.png</normaloff>persons.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCompany">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap>company.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEmblem">
        <property name="pixmap">
         <pixmap>emblem.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="labelLayout">
      <item>
       <widget class="QLabel" name="labelBeschäftigte">
        <property name="text">
         <string>Beschäftigte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelArbeitsmarkt">
        <property name="text">
         <string>Arbeitsmarkt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeView" name="treeViewAngestellte"/>
      </item>
      <item row="0" column="1">
       <widget class="QTreeView" name="treeViewBewerber"/>
      </item>
      <item row="1" column="0">
       <widget class="QTreeView" name="treeViewSoftwareToWrite"/>
      </item>
      <item row="1" column="1">
       <widget class="QTreeView" name="treeViewProjekte"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="treeViewSoftware"/>
    </item>
    <item>
     <widget class="QTextEdit" name="messageEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionNext"/>
   <addaction name="actionNext10"/>
   <addaction name="actionHire"/>
   <addaction name="actionFire"/>
   <addaction name="actionAddSoftware"/>
   <addaction name="actionRemoveProjekt"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="media-seek-forward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next Day</string>
   </property>
   <property name="toolTip">
    <string>Step one Day</string>
   </property>
  </action>
  <action name="actionNext10">
   <property name="icon">
    <iconset theme="media-skip-forward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next 10 Days</string>
   </property>
   <property name="toolTip">
    <string>Step 10 Days</string>
   </property>
  </action>
  <action name="actionHire">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Hire</string>
   </property>
   <property name="toolTip">
    <string>Hire selected</string>
   </property>
  </action>
  <action name="actionFire">
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fire</string>
   </property>
   <property name="toolTip">
    <string>Fire selected</string>
   </property>
  </action>
  <action name="actionAddSoftware">
   <property name="icon">
    <iconset theme="go-up">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add Software</string>
   </property>
   <property name="toolTip">
    <string>Start this Softwareproject</string>
   </property>
  </action>
  <action name="actionRemoveProjekt">
   <property name="icon">
    <iconset theme="go-down">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove Project</string>
   </property>
   <property name="toolTip">
    <string>Abandon Software</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
